docs/examples/basic_tracer/README.rst

```diff
@@ -0,0 +1,37 @@
+Basic Trace
+===========
+
+These examples show how to use OpenTelemetry to create and export Spans.
+
+There are two different examples:
+
+* basic_trace: Shows how to configure a SpanProcessor and Exporter, and how to create a tracer and span.
+
+* resources: Shows how to add resource information to a Provider. Note that this must be run on a GCP instance.
```

Comment:
if only GCE is supported right now, should this be changed to 
```suggestion
* resources: Shows how to add resource information to a Provider. Note that this must be run on a Google Compute Engine instance.
```

Reply:
Excellent suggestion